test/compilable/test6461.sh

```diff
@@ -1,13 +1,12 @@
 #!/usr/bin/env bash
 
-dir=${RESULTS_DIR}/compilable
-src=compilable/extra-files/${TEST_NAME}
+src=${EXTRA_FILES}/${TEST_NAME}
 
-$DMD -lib -m${MODEL} -of${dir}/a${LIBEXT} -I${src} ${src}/a.d || exit 1
-$DMD -lib -m${MODEL} -of${dir}/b${LIBEXT} -I${src} ${src}/b.d || exit 1
+$DMD -lib -m${MODEL} -of${RESULTS_OUTPUT}/a${LIBEXT} -I${src} ${src}/a.d || exit 1
+$DMD -lib -m${MODEL} -of${RESULTS_OUTPUT}/b${LIBEXT} -I${src} ${src}/b.d || exit 1
 
-$DMD -m${MODEL} -od${dir} -I${src} ${src}/main.d ${dir}/a${LIBEXT} ${dir}/b${LIBEXT} || exit 1
+$DMD -m${MODEL} -od${RESULTS_OUTPUT} -I${src} ${src}/main.d ${RESULTS_OUTPUT}/a${LIBEXT} ${RESULTS_OUTPUT}/b${LIBEXT} || exit 1
 
-rm -f ${dir}/{a${LIBEXT} b${LIBEXT} main${EXE} main${OBJ}}
+rm -f ${RESULTS_OUTPUT}/{a${LIBEXT} b${LIBEXT} main${EXE} main${OBJ}}
```

Comment:
That's one of the cases where just nuking the `RESULTS_OUTPUT` folder would make sense (and is probably a good default (if no error happened), but I went with keeping the existing behavior to make reviewing easy.